pkg/client/manifest_client.go

```diff
@@ -22,11 +22,16 @@ type CreateExternalSecretOpts struct {
 // ManifestService implements the business logic
 type ManifestService interface {
 	CreateExternalSecret(ctx context.Context, opts CreateExternalSecretOpts) (*ExternalSecret, error)
+	// I don't like this, but I think ManifestService is an equally big problem
```

Comment:
Why don't you like it?

Reply:
The name `Manifest` was initially derived from the Kubernetes manifest lingo. I am not certain this is conveyed, maybe mirroring something like `Kube` or similar would be better? I think there are things around kube/manifest that could use a proper review

Reply:
Agreed. Manifest confuses me. A KubeService/KubernetesServer would make alot more senses (at least to me)